packages/editor/src/components/profile/Profile.tsx

```diff
@@ -0,0 +1,245 @@
+          </div>
+          <span className="px-2 py-3 text-lg text-white">{name.value}</span>
+        </div>
+        <hr className="mx-4 text-[#A0A1A2] opacity-10" />
+        <div className="px-4 py-2">
+          <div className="grid grid-cols-1 gap-3">
+            <InputGroup
+              className="mr-0 flex flex-row items-center justify-start"
+              label="Invite Code"
+              labelClassName="text-left w-20"
+            >
+              <StringInput
+                type={showKeys.inviteCode.value ? 'text' : 'password'}
+                value={inviteCode.value}
+                endComponent={
+                  <button
+                    type="button"
+                    className="m-0 h-5 w-5 flex-shrink-0 border-none p-0 hover:opacity-80"
+                    onClick={() => toggleKey('inviteCode')}
+                    name="inviteCode"
+                  >
+                    {showKeys.inviteCode.value ? (
+                      <PiEyeBold className="font-small text-[#6B7280]" />
+                    ) : (
+                      <PiEyeClosedBold className="font-small text-[#6B7280]" />
+                    )}
+                  </button>
+                }
+              />
+            </InputGroup>
+            <InputGroup
+              className="mr-0 flex flex-row items-center justify-start"
+              label="User ID"
+              labelClassName="text-left w-20"
+            >
+              <StringInput
+                type={showKeys.userId.value ? 'text' : 'password'}
+                value={userId.value}
+                endComponent={
+                  <button
+                    type="button"
+                    className="m-0 h-5 w-5 flex-shrink-0 border-none p-0 hover:opacity-80"
+                    onClick={() => toggleKey('userId')}
+                    name="userId"
+                  >
+                    {showKeys.userId.value ? (
+                      <PiEyeBold className="font-small text-[#6B7280]" />
+                    ) : (
+                      <PiEyeClosedBold className="font-small text-[#6B7280]" />
+                    )}
+                  </button>
+                }
+              />
+            </InputGroup>
+          </div>
+        </div>
+
+        <hr className="mx-4 text-[#A0A1A2] opacity-10" />
+        <div className="space-y-4 px-4 py-5">
+          <span className="text-md ml-4 flex cursor-pointer items-center gap-2 text-[#A0A1A2]">
+            <LuSettings />{' '}
+            <span className="text-xs" onClick={() => PopupMenuServices.showPopupMenu(UserMenus.Settings)}>
+              Settings
+            </span>
+          </span>
+          <span className="text-md ml-4 flex cursor-pointer items-center gap-2 text-[#A0A1A2]" onClick={handleLogout}>
+            <FaArrowRightToBracket /> <span className="text-xs">Log out</span>
+          </span>
+          <span className="text-md ml-4 flex cursor-pointer items-center gap-2 text-[#A0A1A2]">
```

Comment:
This button doesn't do anything. I don't know if it's supposed to be here - my take would be to make users go to the dashboard to delete their account and not have a delete account button in the studio, but I'm not in charge of UX decisions. If it is supposed to be here, then it needs to do something, and I would imagine there should be a confirmation prompt so a stray click doesn't wipe your account.